myuw/templates/handlebars/card/instructor_schedule/course_resource/class_website.html

```diff
@@ -15,7 +15,7 @@
                     None provided
                 {{else}}
                     {{#if sln}}
-                        <a class="course_website_update" href="javascript:void(0);" data-href="https://sdb.admin.uw.edu/sisMyUWClass/uwnetid/pop/classurl.aspx?quarter={{ quarter }}+{{ year }}&amp;sln={{ sln }}&amp;chanid=11" rel="{{ curriculum_abbr }} {{course_number}} {{section_id}}" data-linklabel="Add Course Website: {{ curriculum_abbr }} {{course_number}} {{section_id}}">Add</a>
+                        <a class="course_website_update" href="javascript:void(0);" data-href="https://sdb.admin.uw.edu/sisMyUWClass/uwnetid/pop/classurl.aspx?quarter={{ quarter }}+{{ year }}&amp;sln={{ sln }}&amp;chanid=11" rel="{{ curriculum_abbr }} {{course_number}} {{section_id}}" data-linklabel="Add Course Website: {{ curriculum_abbr }} {{course_number}} {{section_id}}" title="Course Canvas links are displayed to students by default.">Add</a>
```

Comment:
Can we set the title to `title="Add a website for your course"` and then move the canvas info into the text below? (see below)